src/components/charts.js

```diff
@@ -0,0 +1,237 @@
+import React from 'react';
+import * as d3 from 'd3';
+import {Line, Bar, defaults} from 'react-chartjs-2';
+import moment from 'moment';
+import PropTypes from 'prop-types';
+
+function Charts(props) {
+  const dates = [];
+  const confirmed = [];
+  const recovered = [];
+  const deceased = [];
+
+  defaults.global.elements.line.fill = false;
+
+  defaults.global.tooltips.intersect = false;
+  defaults.global.tooltips.mode = 'nearest';
+  defaults.global.tooltips.position = 'average';
+  defaults.global.tooltips.backgroundColor = 'rgba(255, 255, 255, 0.6)';
+  defaults.global.tooltips.displayColors = false;
+  defaults.global.tooltips.borderColor = '#c62828';
+  defaults.global.tooltips.borderWidth = 1;
+  defaults.global.tooltips.titleFontColor = '#000';
+  defaults.global.tooltips.bodyFontColor = '#000';
+  defaults.global.tooltips.caretPadding = 4;
+  defaults.global.tooltips.intersect = false;
+  defaults.global.tooltips.mode = 'nearest';
+  defaults.global.tooltips.position = 'nearest';
+
+  defaults.global.legend.display = true;
+  defaults.global.legend.position = 'bottom';
+
+  defaults.global.hover.intersect = false;
+
+  if (!props.timeseries || props.timeseries.length == 0) {
+    return <div></div>;
+  }
+
+  props.timeseries.forEach((data, index) => {
```

Comment:
Why not destructure timeseries from props? Then you stop writing props everywhere. It does add any value.

```js
const { timeseries } = props
```
or even better
 
```js
function Charts({ graphOption, mode, timeseries })
```